Makefile

```diff
@@ -128,3 +128,26 @@ setup-env:
 	venv/bin/pip install --upgrade -r build_calicoctl/requirements.txt
 	@echo "run\n. venv/bin/activate"
 
+
+run-etcd-secure:
```

Comment:
Yup, will move them. Was doing this for easy access to the targets when modifying via command line.